### ISSUE TITLE
Some options I want to see for 0.9.9.23

### ISSUE THREAD

**Issue by ThePlayingGoblin** [ImAiiR/QobuzDownloaderX#69]
So I've been using this tool quite a bit and I would like to address some of the things that I would like to see having an option to change or remove.

I would like to have a toggle to download all label releases into the same folder instead of creating artist folders, and having the URL link tail and Bit-Depth/Sample rate subfolder to be toggleable, and/or the Bit-Depth/Sample rate to be on the same folder. For the duplicate scenario, either skipping these releases or including the url tail for duplicates only would be a neat feature too.

Basically, these are just options that I would like to see, and would help me save a lot of time when changing tags and structuring releases.
